### PR TITLE
noelle update

### PR DESCRIPTION
**Pull request by Noelle-KH**
hiiiiiiiiiiiiiii

### CODE REVIEW

File: controllers/todoController.js [hunk 1]
```diff
@@ -39,7 +41,9 @@ let todoController = {
           .then((todo) => {
             return res.redirect('/todos')
           })
+          .catch(error => console.log(error))
       })
+      .catch(error => console.log(error))
```
Comment (Lilynews) [inline controllers/todoController.js:46]: 1018 checked

Comment (Lilynews) [reply controllers/todoController.js:46]: 1019 checked-start a review

Comment (Lilynews) [reply controllers/todoController.js:46]: yooooo

Comment (Noelle-KH) [reply controllers/todoController.js:46]: 1119 checked～～

Comment (Lilynews) [reply controllers/todoController.js:46]: 1020 checked

**Review by Lilynews — COMMENTED** [review]
1020

File: controllers/todoController.js (hunk 1 above)
Comment (Lilynews) [reply controllers/todoController.js:46]: approve?

**Review by Lilynews — APPROVED** [review]

**Pull request merged**